cmd/debugger/main.go

```diff
@@ -54,21 +55,36 @@ func interactiveMode(remoteConsoleAddr string) {
 	}
 }
 
-func main() {
-	remoteConsoleAddr := os.Getenv("EARTHLY_REMOTE_CONSOLE_ADDR")
-	if remoteConsoleAddr == "" {
-		remoteConsoleAddr = "127.0.0.1:8543"
+func getRemoteDebuggerAddr() string {
+	remoteConsoleAddr, err := ioutil.ReadFile("/run/secrets/earthly_remote_console_addr")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Warning: failed to read earthly_remote_console_addr: %v", err)
```

Comment:
There's a lot of fprintf for errors scattered around. Perhaps it's better that all errors are bubbled up to main and then there's only one place in main that prints errors.

Reply:
I think that's a good point if we were immediately exiting after the errors, but in this instance we're treating it as a warning and not returning an error.

And the other cases are for a message before we drop into the interactive shell.